shoop/addons/admin_module/views/reload.py

```diff
@@ -56,4 +64,8 @@ def form_valid(self, form):
     def get(self, request, *args, **kwargs):
         if request.GET.get("ping"):
             return JsonResponse({"pong": time.time()})
+        elif request.GET.get("migrate"):
+            return JsonResponse({"message": run_command("migrate")})
+        elif request.GET.get("collectstatic"):
+            return JsonResponse({"message": run_command("collectstatic")})
```

Comment:
IMHO all these operations (pip install && migrate && collectstatic) should happen in a single request. Splitting the process to multiple requests risks the addon installation being left in a state where migrations or collectstatic is not ran when some sort of network or web browser error occurs in the middle of installation.

What was the reason to split it to multiple requests? Maybe I'm missing something, because I can't really understand why it's done  this way, since doing it in single request should be simpler too, or is it?